src/course/en/course.json

```diff
@@ -35,17 +35,15 @@
             "_accordion": {
                 "ariaRegion": "This component is an accordion comprised of collapsible content panels containing display text. Select the item titles to toggle the visibility of these content panels."
             },
-            "_blank": {
-                "ariaRegion": ""
-            },
             "_gmcq": {
                 "ariaRegion": "This component is a graphical multiple choice question. Once you have selected an option select the submit button below."
             },
             "_graphic": {
                 "ariaRegion": ""
             },
-            "_hotGraphic": {
-                "ariaRegion": "Below is a component which allows you to select hot spots over an image. Select a hot spot to trigger a popup that includes an image with display text. Select the close button to close the popup."
+            "_hotgraphic": {
+                "ariaRegion": "Below is a component which allows you to select hot spots over an image. Select a hot spot to trigger a popup that includes an image with display text. Select the close button to close the popup.",
+                "ariaPoupupLabel": "Hotgraphic popup"
```

Comment:
Keen spot! I didn't actually type anything – everything in this PR was pulled automatically from schemas. This will then need fixing: https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/master/properties.schema#L15.